ReactWindows/ReactNative.Net46.Tests/Modules/Storage/AsyncStorageModuleTests.cs

```diff
@@ -0,0 +1,609 @@
+﻿using NUnit.Framework;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using ReactNative.Modules.Storage;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+namespace ReactNative.Tests.Modules.Storage
+{
+    [TestFixture, Apartment(ApartmentState.STA)]
+    public class AsyncStorageModuleTests
```

Comment:
Yes, [here](https://github.com/ReactWindows/react-native-windows/blob/master/ReactWindows/ReactNative.Tests/Modules/Storage/AsyncStorageModuleTests.cs). Actually, this test is copy of UWP test. Just by @matthargett request, I rewrote assertion to `Assert.That` expression but the whole idea remains the same.